src/glum/_glm.py

```diff
@@ -1274,6 +1294,7 @@ def predict(
             copy=self._should_copy_X(),
             ensure_2d=True,
             allow_nd=False,
+            drop_first=True,
```

Comment:
```suggestion
            drop_first=self.drop_first,
```

Can you otherwise explain why drop_first should be enforced here?